.github/workflows/docker-build-release.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Update image in GHCR
+
+run-name: >
+  ${{
+    github.event_name == 'delete' && format(
+      'Delete `{0}{1}`',
+      github.event.ref_type == 'branch' && 'branch-' || '',
+      github.event.ref
+    )
+    || github.ref == 'refs/heads/main' && 'Update `dev`'
+      || format(
+        'Update `{0}{1}`',
+        !startsWith(github.ref, 'refs/tags') && 'branch-' || '',
+        github.ref_name
+      )
+  }} docker tag
```

Comment:
What does `run-name` do？

Reply:
<img width="668" alt="Screenshot 2024-08-16 at 2 23 39 PM" src="https://github.com/user-attachments/assets/74fb03b0-6318-42e7-a60e-60425a71e18c">

It controls the run's name in the "Actions" view - the screenshot above is an example. You can find some of my complaints about how irritating it is that the `run-name` must be so complex in the [original pull request](https://github.com/uclahs-cds/tool-Docker-action/pull/14).

Reply:
Ah that's cool!